### PR TITLE
Convert to jest, update dependencies

### PR DESCRIPTION
**Pull request by ehoogerbeets**

### CODE REVIEW

File: scripts/genlikelyloc.js [hunk 1]
```diff
@@ -229,16 +229,70 @@ function getAncestors(region) {
             }
         });
     }
-    
-    containmentReverse[region] = toArray(set);
+
+    containmentReverse[region] = toArray(set).reverse();
```
Comment (gouniLee) [inline scripts/genlikelyloc.js:233]: I think iLib should apply these changes as well. isn't it?

File: src/localematch.js [hunk 2]
```diff
@@ -24217,615 +24217,615 @@ export const matchdata = {
...
+            "154",
             "150",
-            "154"
+            "001"
         ],
         "CR": [
-            "001",
-            "UN",
-            "019",
+            "013",
```
Comment (gouniLee) [inline src/localematch.js:24679]: the order of these are important?

Comment (ehoogerbeets) [reply src/localematch.js:24679]: The order is important, yes. It takes the first match

File: scripts/genlikelyloc.js (hunk 1 above)
Comment (ehoogerbeets) [reply scripts/genlikelyloc.js:233]: probably yes

**Comment by ehoogerbeets** [comment]
@gouniLee ready for re-review

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,37 +1,63 @@
...
 workflows:
   version: 2
   test-all-node-versions:
     jobs:
-      - test:
-          docker_image: circleci/node:8-browsers
       - test:
           docker_image: circleci/node:10-browsers
-      - test:
+      - testold:
```
Comment (gouniLee) [inline .circleci/config.yml:53]: why testing node v12 is `testold`?

Comment (ehoogerbeets) [reply .circleci/config.yml:53]: v12 does not have any ESM module support or the browser tools

**Review by gouniLee — APPROVED** [review]

**Pull request merged**